Gemfile.jruby-1.9.lock

```diff
@@ -8,14 +8,16 @@ PATH
       filesize (= 0.0.4)
```

Comment:
why not? I mean, if plugins need to be updated across updates, then necessarily their dependencies will also be updated...